### DISCUSSION TITLE
Any way to re-register the command into server as soon as possible

### DISCUSSION THREAD

**Discussion by ThanatosDi** [Pycord-Development/pycord#1022] [Q&A] [answered]
English is not my native language but I try my best.

For example, now I build a bot, and I add a new slash command.

Now I need kick the bot out my server and reinvite it into server. 

The new slash command will appear.

Any way can add register slash command into server as soon as possible except kick the bot ?

**Comment by middledot** [comment, marked as answer]
Kicking and reinviting the bot is the only way (as far as I can tell) to force slash commands to become up to date. Otherwise, you need to wait around an hour for them to re-register.
